Lab_2.html

```diff
@@ -13647,7 +13647,7 @@
 </div><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <p>Version 2) statistical question: " given the same background distribution, but with 10,000 trials instead of one, how does the probability and sigma of the hypothetical measuremnt change with the added trials.?"</p>
-<p>As calculated above, with the background ive used applied over 10000 trials, a signal of 18.2 no longer has an assosiated sigma value of 5... Despite # of trials being the only change, I can no longer claim a discovery.</p>
+<p>As calculated above, with the background ive used applied over 10000 trials, a signal of 18.2 no longer has an associated sigma value of 5 but rather 3.2178. Despite 10k trials being the only change, I can no longer claim a discovery.</p>
```

Comment:
more spelling and I think this is what you were going for here. You do it most of the time but I added what value of sigma that you got. She mentioned in my feedback that doing this is something that she wants to see